packages/reserve-site/src/pages/index.tsx

```diff
@@ -0,0 +1,147 @@
+import { Addresses, HoldingsData } from 'src/service/Data'
+
+interface ContentShape {
+  title: string
+}
+
+interface Props {
+  INTRO: FrontMatterResult<ContentShape>
+  INITIAL_TARGET: FrontMatterResult<ContentShape>
+  ABOUT: FrontMatterResult<ContentShape>
+  ATTESTATIONS: FrontMatterResult<ContentShape>
+}
+
+export default function Home(props: HoldingsData & Addresses & Props) {
+  return (
+    <>
+      <Head />
+      <div css={rootStyle}>
+        <div css={containerStyle}>
+          <NavBar />
+          <main css={mainStyle}>
+            <Section title={props.INTRO.attributes.title} content={props.INTRO.body} />
+            <Section
+              title={'Current Reserve Holdings'}
+              subHeading={<Updated date={props.updatedDate} />}
+            >
+              <Holdings
+                total={props.total}
+                inCustody={props.inCustody}
+                onChain={props.onChain}
+                cUSD={props.cUSD}
+                DAI={props.DAI}
+                BTC={props.BTC}
+                ETH={props.ETH}
+                ratio={props.ratio}
+              />
+            </Section>
+            <Section title="Stable Value Assets">
+              <StableTokens cUSD={props.cUSD} />
+            </Section>
+            <Section title="Reserve Ratio">
+              <Ratios total={props.ratio} unfrozen={props.unFrozenRatio} />
+            </Section>
+            <Section title={'Reserve Addresses'}>
+              <ReserveAddresses
+                dai={props.daiAddress}
+                btc={props.btcAddress}
+                eth={props.ethAddress}
+                celo={props.celoAddress}
+                custody={props.custodyAddress}
+              />
+            </Section>
+            <Section
+              title={props.INITIAL_TARGET.attributes.title}
+              content={props.INITIAL_TARGET.body}
+            >
+              <TargetGraph />
+            </Section>
+
+            <Section title={props.ABOUT.attributes.title} content={props.ABOUT.body} />
+            <Section
+              title={props.ATTESTATIONS.attributes.title}
+              content={props.ATTESTATIONS.body}
+            />
+          </main>
+        </div>
+        <Footer />
+      </div>
+    </>
+  )
+}
+
+function Updated({ date }) {
+  return (
+    <small css={dateStyle}>
+      <strong>Updated </strong>
+      {new Date(date).toLocaleDateString('default', {
+        month: 'short',
+        day: 'numeric',
+        year: 'numeric',
+      })}
+    </small>
+  )
+}
+
+const rootStyle = css({
+  display: 'flex',
+  flexDirection: 'column',
+  minHeight: '100vh',
+  flex: 1,
+  alignItems: 'center',
+  justifyContent: ' space-between',
+})
+
+const mainStyle = css({
+  width: '100%',
+  maxWidth: 960,
+})
+
+const dateStyle = css({ marginBottom: 36, display: 'block' })
+
+const containerStyle = css(flexCol, { flex: 1, width: '100%', alignItems: 'center' })
+
+export async function getStaticProps() {
```

Comment:
Where is this used?

Reply:
its a next.js method that is called at build time and then if  the `unstable_revalidate` is defined at most once every X seconds to get new props on server.